Code/Engine/RendererFoundation/RendererFoundationDLL.h

```diff
@@ -81,8 +81,9 @@ struct EZ_RENDERERFOUNDATION_DLL ezGALIndexType
 {
   enum Enum
   {
-    UShort,
-    UInt,
+    None,   // indices are not used, vertices are just used in order to form primitives
```

Comment:
Why is the 'None' option needed here? Typically we just use an invalid index buffer if we don't want indexed rendering.

Reply:
Because the enum is used in the creation of the dynamic buffer resource (and thus also the component) and I didn't want to add another enum that would be mostly redundant.
'None' is used to set up meshes that don't use indexed rendering.